.golangci.yaml

```diff
@@ -38,6 +38,16 @@ run:
   # If it's not please let us know.
   # "/" will be replaced by current OS file path separator to properly work on Windows.
   skip-files:
+    - "redfish/provider/data_source_redfish_bios.go"
+    - "redfish/provider/data_source_redfish_firmware_inventory.go"
+    - "redfish/provider/data_source_redfish_storage.go"
+    - "redfish/provider/data_source_redfish_system_boot.go"
+    - "redfish/provider/data_source_redfish_virtual_media.go"
+    - "redfish/provider/resource_redfish_storage_volume.go"
+    - "redfish/provider/resource_redfish_storage_volume.go"
+    - "redfish/provider/resource_redfish_user_account.go"
```

Comment:
Can you remove user account and virtual media since the PRs are already up for both of these resources.

Reply:
done